addons/kibana/6.8.x/kibana-6.yaml

```diff
@@ -73,6 +73,27 @@ spec:
             value: "kubeaddons"
           - name: "KIBANA_SERVICE_NAME"
             value: "kibana-kubeaddons"
+        - name: extra-index-patterns
+          image: curlimages/curl
+          command:
+            - sh
+            - -c
+            - |
+              #!/bin/sh
+              KB_URL=http://localhost:5601
+              while [[ "$(curl -s -o /dev/null -w '%{http_code}\n' -L $KB_URL/api/status)" != "200" ]]; do sleep 1; done
+              # Create index patterns
+              if [[ "$(curl -s -o /dev/null -w '%{http_code}\n' -L $KB_URL/api/saved_objects/index-pattern/kubernetes_audit)" != "200" ]]; then
+                curl -XPOST "$KB_URL/api/saved_objects/index-pattern/kubernetes_audit" \
+                  -H "Content-Type: application/json; charset=utf-8" -H 'kbn-xsrf: true' \
+                  -d '{"attributes":{"title":"kubernetes_audit-*","timeFieldName":"@ts","fields":"[]"}}'
+              fi
+              if [[ "$(curl -s -o /dev/null -w '%{http_code}\n' -L $KB_URL/api/saved_objects/index-pattern/kubernetes_host)" != "200" ]]; then
+                curl -XPOST "$KB_URL/api/saved_objects/index-pattern/kubernetes_host" \
+                  -H "Content-Type: application/json; charset=utf-8" -H 'kbn-xsrf: true' \
+                  -d '{"attributes":{"title":"kubernetes_host-*","timeFieldName":"@ts","fields":"[]"}}'
+              fi
+              sleep infinity
```

Comment:
you want the shell to run forever?

Reply:
yes. otherwise it will end. If the container ends, since it's a sidecar container instead of an initContainer, the pod will end and it will be restarted causing a crash loop backoff.

Reply:
This is just for now. 
I plan to update the addons to use upstream charts from elastic directly and use lifecycle PostStartHook